libs/react-components/src/lib/components/Typography/styles.module.scss

```diff
@@ -94,6 +95,21 @@
   }
 }
 
+.eyebrow2 {
```

Comment:
Since the only change in eyebrow and eyebrow2 classes is `align-items`, instead of adding an eyebrow2 class, we can maybe use a css variable to set the `align-items` property:

```
.eyebrow {
  align-items: var(--td-web-typography-eyebrow-alignment, 'center');
  display: flex;
  flex-direction: column;
  gap: 8px;
  &::after {
    background-color: var(--cv-theme-tertiary);
    border-radius: 2px;
    content: '';
    height: 2px;
    width: 20px;
  }
}
```

This css property can be set in any app that uses this component.
```
--td-web-typography-eyebrow-alignment: left;
```

Reply:
Hi Sahitya.
Thanks for the suggestion. 

I choose to add the diffrent options in a dropdown y the alignment as a parameter for the eyebrow.
Let me know what you think.

![image](https://github.com/user-attachments/assets/ab8a579d-3ae4-4f20-9f47-8faf35f5ddcd)